index.html

```diff
@@ -608,6 +614,18 @@ <h2>Invoke idle callback timeout algorithm</h2>
         </li>
       </ol>
     </section>
+    <section>
+      <h2>Is high-priority work pending algorithm</h2>
+      <p>The <dfn>is high-priority work pending algorithm</dfn>:</p>
+      <ol>
+        <li>If <var>event_loop</var> has any task queues that contain a 
+	<a>task</a> <a>queued</a> with the <a>user interaction task source</a>,
```

Comment:
We need to restrict this to [*runnable* tasks](https://html.spec.whatwg.org/multipage/webappapis.html#concept-task-runnable).

Each [task](https://html.spec.whatwg.org/multipage/webappapis.html#concept-task) is a struct which has a source as its member so there is no need to say a task queue contains a task queued with a particular task source. use say a task queue contains a task with a particular source; e.g. if <var>event loop</var> has any task queues with a runnable task with <a>user interaction task source</a>.

But do we really want to restrict this to just "user interaction task source"? Networking tasks are probably important and so are tasks with [media element event ask sources](https://html.spec.whatwg.org/multipage/media.html#media-element-event-task-source).

Reply:
Done